exporters/BxDRobotExporter/BxDRobotExporter/BxDRobotExporter/Wizard/Components/DefinePartPanel.cs

```diff
@@ -43,7 +43,7 @@ public DefinePartPanel(RigidNode_Base node)
             PortOneUpDown.Minimum = 3;
             PortTwoUpDown.Minimum = 3;
 
-            PortOneUpDown.Value = WizardData.Instance.NextFreePort;
+            PortOneUpDown.Value = WizardData.Instance.NextFreePort < 19 ? WizardData.Instance.NextFreePort : 19;
```

Comment:
This seems like you are saying as long as the port is less than 19 its the port, but if its over 19 then it will be 19 no matter what .
so
 
`50 < 19 ? 50 : 19` is **19** and `40 < 19 ? 40 : 19` is **19**